docs/topics/core_concepts.md

```diff
@@ -18,12 +18,14 @@ An agent operates within an environment that includes interactions with several
 | Components | Description | Can be Local | Can be Remote | Related Protocols |
 |--|--|--|--|--|
 | User | Interface with the user, either directly or indirectly | ✗ | ✓ | |
```

Comment:
@mbrg  - why user can't be local?
again thinking about agents running locally such as SDK built agents.